src/coord_seq.rs

```diff
@@ -162,6 +339,42 @@ impl<'a> CoordSeq<'a> {
         }
     }
 
+    /// Returns the number of lines of the `CoordSeq` object.
+    ///
+    /// Note: This is an alias to the [`size`](#method.size) method.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use geos::{CoordDimensions, CoordSeq};
+    ///
+    /// let coords = CoordSeq::new(2, CoordDimensions::ThreeD)
+    ///                       .expect("failed to create CoordSeq");
+    /// assert!(coords.number_of_lines() == Ok(2));
+    ///
+    /// let coords = CoordSeq::new_from_vec(&[&[1.], &[2.], &[3.], &[4.]])
+    ///                       .expect("failed to create CoordSeq");
+    /// assert!(coords.number_of_lines() == Ok(4));
+    /// ```
+    pub fn number_of_lines(&self) -> GResult<usize> {
```

Comment:
not sure about this `line` naming.
line can be quite a lot of things in the `geo` context.

why don't you like `size()` (and `idx` instead of `line` in the signature/comments) ?

Reply:
Because you can see the `CoordSequence` as a matrix, therefore it's easier to speak about lines rather than "index" which is very vague.